### PR TITLE
List pytest-doctestplus in doctest docs

### PR DESCRIPTION
**Pull request by nicoddemus**
As per https://github.com/pytest-dev/pytest/discussions/8088

### CODE REVIEW

**Comment by nicoddemus** [comment]
cc @pllim

File: doc/en/doctest.rst [hunk 1]
```diff
@@ -303,3 +294,18 @@ docstring.
...
+
+
+Alternatives
+------------
+
+While the built-in pytest support provides a good set of functionalities for using
+doctests, if you use them extensively you might be interested in those external packages
+which add many more features, and include pytest integration:
+
+* `pytest-doctestplus <https://github.com/astropy/pytest-doctestplus>`__: provides
```
Comment (pllim) [inline doc/en/doctest.rst:306]: @astrofrog and @saimn , any objection to this mention here?

Comment (saimn) [reply doc/en/doctest.rst:306]: No, sounds good!

**Review by bluetech — APPROVED** [review]

**Comment by pllim** [comment]
Thank you!

**Pull request merged**